directories/src/redux/actions/items-actions.js

```diff
@@ -0,0 +1,45 @@
+import {
+  API_FETCH_ITEMS,
+  API_GIVE_PEER_ID,
+  API_START_TRANSACTION,
+  API_SELECT_NEED
+} from './action-types';
+
+import { 
+  requestFetchItems,
+  requestGivePeerId,
+  requestStartTransaction,
+  requestSelectNeed 
+} from '../services';
+
+export const fetchItemsRequest = (needle = '') => {
+  const request = requestFetchItems(needle);
```

Comment:
this stuff belongs in an epic. we want the actions to be just an object, we don't want to mix up the API fetching in it